bundled/muparser_v2_2_4/src/muParserTokenReader.cpp

```diff
@@ -902,7 +902,7 @@ namespace mu
     std::size_t iEnd(0), iSkip(0);
 
     // parser over escaped '\"' end replace them with '"'
-    for(iEnd=(int)strBuf.find( _T("\"") ); iEnd!=0 && iEnd!=string_type::npos; iEnd=(int)strBuf.find( _T("\""), iEnd))
+    for(iEnd=(int)strBuf.find( _T('\"') ); iEnd!=0 && iEnd!=string_type::npos; iEnd=(int)strBuf.find( _T('\"'), iEnd))

```

Comment:
Out of curiosity, why do you still need the backslash?

Reply:
Probably, we don't. I would expect it to be exactly the same.

Reply:
https://godbolt.org/g/hg15M2 creates the same assembly.